Postman/Postman.js

```diff
@@ -1,54 +1,49 @@
-"use strict";
+const Q = require('q');
+const _ = require('underscore');
+const moment = require('moment-timezone');
 
-var Q = require('q');
-var _ = require('underscore');
-var moment = require('moment-timezone');
-
-var MailToUser = require('Postman/lib/MailToUser');
-var Librarian = require('Librarian/Librarian');
-var log = require('common/log');
-var PERIODICAL_DAY = require('common/constant').PERIODICAL_DAY;
+const MailToUser = require('Postman/lib/MailToUser');
+const Librarian = require('Librarian/Librarian');
+const log = require('common/log');
 
 
 /**
 @constructor
 **/
-function Postman(){
-	this.conditions = {
-		isVerified: true
-	};
-	this.Mailer = require('common/Mailer')();
-	this.Utils = require('common/Utils')();
-	this.User = require('models/User');
-	this.Series = require('models/Series');
-	this.users = [];
-	this.Librarian = new Librarian();
-	this._defer = this.Librarian.defer;
-
-	return this;
+function Postman() {
+  this.conditions = {
+    isVerified: true,
+  };
+  this.Mailer = require('common/Mailer')();
+  this.Utils = require('common/Utils')();
+  this.User = require('models/User');
+  this.Series = require('models/Series');
+  this.users = [];
+  this.Librarian = new Librarian();
+  this._defer = this.Librarian.defer;
 }
 
 
-Postman.prototype.fetch = function(done){
-	var _self = this;
-	this.User.find(this.conditions, function(err, users){
-		if(err){
-			return done(err);
-		}
-		_self.users = users;
-		done(null, users);
-	});
+Postman.prototype.fetch = function fetch(done) {
+  const _self = this;
+  this.User.find(this.conditions, (err, users)=> {
+    if (err) {
+      return done(err);
+    }
+    _self.users = users;
+    done(null, users);
+  });
 };
 
 
-Postman.prototype.sent = function(done){
-	Q.all(this.users.map(MailToUser.send))
-	.then(function(){
-		done();
-	})
-	.fail(function(err){
-		done(null, err);
-	});
+Postman.prototype.sent = function sent(done) {
+  Q.all(this.users.map(MailToUser.send))
+  .then(()=> {
+    return done();
+  })
+  .fail((err)=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).